tests/baselines/reference/quickInfoLink5.baseline

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/quickInfoLink5.ts",
+      "position": 61,
+      "name": ""
+    },
+    "quickInfo": {
+      "kind": "const",
+      "kindModifiers": "",
+      "textSpan": {
+        "start": 61,
+        "length": 1
+      },
+      "displayParts": [
+        {
+          "text": "const",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "B",
+          "kind": "localName"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "456",
+          "kind": "stringLiteral"
+        }
+      ],
+      "documentation": [
+        {
+          "text": "See ",
+          "kind": "text"
+        },
+        {
+          "text": "{@link ",
+          "kind": "link"
+        },
+        {
+          "text": "}",
+          "kind": "link"
```

Comment:
this is an OK result but I'd prefer not to change the formatting of a malformed or unresolved link tag.

Reply:
Do you mean the result should contain empty `linkText` for this case?
```json
{
    "text": "",
    "kind": "linkText"
},
```